### PR TITLE
add build logs to e2e DumpInfo

### PR DESCRIPTION
**Pull request by asalkeld**
relates #1481

```release-note
NONE
```

### CODE REVIEW

**Comment by mjudeikis** [comment]
Inspecting

**Comment by mjudeikis** [comment]
/retest 
"node going away issue". email sent

File: test/clients/openshift/dumpinfo.go [hunk 1]
```diff
@@ -22,6 +22,42 @@ func (cli *Client) DumpInfo(namespace string) error {
...
+		var err error
+		f, err = os.Create(filepath.Join(*artifactDir, fmt.Sprintf("builds-%s.yaml", namespace)))
+		if err != nil {
+			logrus.Warn(err)
+		} else {
+			defer f.Close()
+		}
+	}
+
+	list, err := cli.BuildV1.Builds(namespace).List(metav1.ListOptions{})
```
Comment (jim-minter) [inline test/clients/openshift/dumpinfo.go]: get the build logs as well

**Pull request merged**